packages/connect-popup/e2e/tests/popup-close.test.ts

```diff
@@ -434,6 +434,11 @@ test('popup should behave properly with subsequent calls', async ({ page, contex
     popupClosedPromise = new Promise(resolve => {
         popup.on('close', () => resolve(undefined));
     });
+
+    log(
+        'aha, this is probably expected to close automatically. but it does not. the reason is that this one already opens getPubKey popup, not getFeatures. getFetures seems to be handled without popup',
+    );
```

Comment:
UX-wise, it looks good. Can't it be that I already have permissions saved from a previous test so I don't need to accept permissions? 
I can't recall what recent change might have caused this.

Reply:
I don't exactly know what change affected this. I was looking at this yesterday with Marek and on local the behavior was the same as the test - eg. the popup did show up for getFeatures, but it closes itself quickly since it's not actually needed. But in CI the popup doesn't even open.